### ISSUE TITLE
I/O logging error during test suite

### ISSUE THREAD

**Issue by TomNicholas** [tomwhite/cubed#61]
Hey, I finally got around to cloning this and I tried running the tests. They all passed (woo!), but I also got many of these logging warnings

```
--- Logging error ---
Traceback (most recent call last):
  File "/home/tom/miniconda3/envs/cubed-test/lib/python3.9/logging/__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
ValueError: I/O operation on closed file.
Call stack:
  File "/home/tom/miniconda3/envs/cubed-test/lib/python3.9/site-packages/lithops/executors.py", line 602, in clean
    logger.info(f'ExecutorID {self.executor_id} - Cleaning temporary data')
Message: 'ExecutorID 06c52f-51 - Cleaning temporary data'
Arguments: ()
```

(This message printed many times but pytest counted it as 1 warning.)

**Is this a problem?** Seems like it might be lithops' fault.

The conda .yml file I used to install the dependencies (in a fresh empty conda env) was this:

```
name: cubed-test

channels:
  - conda-forge

dependencies:
  - apache-beam
  - dask[array]
  - fsspec
  - networkx < 2.8.3 # https://github.com/networkx/networkx/pull/5667
  - numpy >= 1.22
  - pydot
  - pytest
  - pytest-cov
  - pytest-mock
  - pip
  - rechunker
  - s3fs
  - tenacity
  - tqdm
  - zarr < 2.12
  - pip:
    -  git+https://github.com/lithops-cloud/lithops.git#egg=lithops[aws]
```

**Comment by TomNicholas** [comment]
(Okay I didn't follow the correct install instructions, ignore this)

**Comment by tomwhite** [comment]
Thanks for opening this @TomNicholas. I see these warnings too - they seem to be harmless, but should be looked at at some point.

**Comment by tomwhite** [comment]
Thanks for opening this @TomNicholas. I see these warnings too - they seem to be harmless, but should be looked at at some point.

**Comment by tomwhite** [comment]
I've created a fix for this in #105
